test/LocalNeighborhood/test_ln.py

```diff
@@ -44,3 +46,51 @@ def test_format_error(self):
                                output_file=OUT_FILE)
 
     # Write tests for the Local Neighborhood run function here
+    """
+    Run test for the Local Neighborhood run function
+    """
+    def test_localneighborhood_required(self):
```

Comment:
I believe this test is failing because the Docker image is not available for GitHub to pull from DockerHub. When I run locally I get the error
```
404 Client Error for http+docker://localnpipe/v1.41/images/create?tag=latest&fromImage=otjohnson%2Flocal-neighborhood: Not Found ("pull access denied for otjohnson/local-neighborhood, repository does not exist or may require 'docker login': denied: requested access to the resource is denied")
```
We could decide to ignore that if the tests all pass locally.

Reply:
If the tests pass for you locally, I'm satisfied that it works. Let's focus on DOMINO